scout/constants/panels.py

```diff
@@ -9,3 +9,9 @@
     ("custom_inheritance_models", "custom_inheritance_models"),
     ("comment", "comment"),
 ]
+
+PANELAPP_CONFIDENCE_EXCLUDE = {
```

Comment:
This has changed, and I'm moving it here from gene_tags, because it's panel-related

Reply:
Yikes - well spotted! Only on the new API I hope?

Reply:
Yes, only in the API!